samples/ImageScanning/cs/ImageScanningSample_NetCore3/Program.cs

```diff
@@ -0,0 +1,252 @@
+﻿// Copyright (C) Microsoft Corporation. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using Windows.Media;
+using Microsoft.AI.Skills.Vision.ImageScanningPreview;
+using Windows.Graphics.Imaging;
+using Windows.Storage;
+using Windows.Storage.Streams;
+
+namespace ImageScanningSample_NetCore3
+{
+    class Program
+    {
+        //
+        // Load a VideoFrame from a specified image file path
+        //
+        static async Task<VideoFrame> LoadVideoFrameFromImageFileAsync(string imageFilePath)
+        {
+            VideoFrame resultFrame = null;
+            
+            try
+            {
+                StorageFile file = await StorageFile.GetFileFromPathAsync(imageFilePath);
+                var fileExtension = file.FileType;
+                if (fileExtension != ".jpg" && fileExtension != ".png")
+                {
+                    throw new ArgumentException("This app parses only .png and .jpg image files");
+                }
+
+                using (IRandomAccessStream stream = await file.OpenAsync(FileAccessMode.Read))
+                {
+                    SoftwareBitmap softwareBitmap = null;
+
+                    // Create the decoder from the stream 
+                    BitmapDecoder decoder = await BitmapDecoder.CreateAsync(stream);
+
+                    // Get the SoftwareBitmap representation of the file in BGRA8 format
+                    softwareBitmap = await decoder.GetSoftwareBitmapAsync();
+
+                    // Convert to friendly format for UI display purpose
+                    softwareBitmap = SoftwareBitmap.Convert(softwareBitmap, BitmapPixelFormat.Bgra8, BitmapAlphaMode.Premultiplied);
+
+                    // Encapsulate the image in a VideoFrame instance
+                    resultFrame = VideoFrame.CreateWithSoftwareBitmap(softwareBitmap);
+                }
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine($"Could not load VideoFrame from file: {imageFilePath}");
+                throw ex;
+            }
+            return resultFrame;
+        }
+
+        //
+        // Save a modified VideoFrame using an existing image file path with an appended suffix
+        //
+        static async Task<string> SaveModifiedVideoFrameToFileAsync(string imageFilePath, VideoFrame frame)
+        {
+            string fileName = imageFilePath;
```

Comment:
nit: Same comment, you can remove this and just return `modifiedFile.Path` from inside the `try` block